example/pubspec.lock

```diff
@@ -218,7 +218,7 @@ packages:
       path: ".."
       relative: true
     source: path
-    version: "1.0.1"
+    version: "2.0.0"
```

Comment:
Let's be consistent with version numbers. Differentiate between major and minor release.